### ISSUE TITLE
Set a timeout when a dialog appears

### ISSUE THREAD

**Issue by pkillnine** [qutebrowser/qutebrowser#6136]
I may be typing, and then a dialog box appears, and it takes my typing to be a response to it.

It would be nice to have a countdown in the box too.

**Comment by The-Compiler** [comment]
Can you elaborate a bit on what kind of dialogs those are? Something like `hints.auto_follow_timeout` but for those dialogs might be one possible solution, but I wonder if there are others (like, for permissions, perhaps a totally different UI).

**Comment by pkillnine** [comment]
@The-Compiler I mean for permissions, yea. I'll be typing and then a website asks for permission for notifications I think. But that style of dialog where you press keys to respond.

**Comment by The-Compiler** [comment]
I don't think this is much of an issue for authentication prompts. You'd mostly end up typing an username or password, which you can easily delete again (rather than accidentally accepting a prompt).

**Comment by The-Compiler** [comment]
Another use case where this can be annoying (and an in-page UI might be better perhaps?) is certificate errors, see [DOS by notifications : qutebrowser](https://old.reddit.com/r/qutebrowser/comments/1hcy9yj/dos_by_notifications/).

**Comment by The-Compiler** [comment]
Another consideration, from #8774:

> after dismissal, the text entered while in command mode completely disappears showing only the status bar.
